change/@fluentui-web-components-12ada8cd-78a1-443f-8c95-c1d01e1f2b24.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Not sure what happened but there were a lot of changes in `packages/web-components/docs/api-report.md` that don't relate to my PR 🤔

Reply:
You should revert the changes in that file, and delete this change file for web-components, since it's unrelated to this PR.